src/csv/CHERI_CSR.csv

```diff
@@ -30,4 +30,4 @@ direct write if address didn't change","✔","","","✔","✔","Zcmt","Jump Vect
 "stdc","0x163","","","S","SRW, <<asr_perm>>","<<null-cap>>","","","","","","✔","","S-mode","Supervisor Trap Data Capability (scratch register)","","","","","","","","","","","","","","","","","","","","",""
 "ddc","0x416","","","U","URW","<<infinite-cap>>","","","","","","✔","","none","User Default Data Capability","","","","","","","","","","","","","","","","","","","","",""
 "pcc","0xcb0","","","U","URO","<<infinite-cap>>
-(address = boot address)","","","✔","","","✔","✔","none","User Program Counter Capability (to allow reading in legacy mode)","","","","","","","","","","","","","","","","","","","","",""
\ No newline at end of file
+(address = boot address)","","","✔","","","✔","✔","none","User Program Counter Capability (to allow reading in legacy mode)","","","","","","","","","","","","","","","","","","","","",""
```

Comment:
These files are downloaded from a spreadsheet in Google Drive. Did you modify the spreadsheet or the CSVs?

Reply:
I just ran pre-commit so it edited the checked in csvs. I can exclude these from the check or modify the download step to add a newline

Reply:
Thanks! Either that, modify the download or modify the spreadsheet. Otherwise, we will continuously get this change when we trigger an update of the CSV.